src/api/systems.tsx

```diff
@@ -1,364 +1,363 @@
-import {

```

Comment:
I think you may have changed the line endings from CRLF to LF - I will update it in one of mine, but could you put it back here so it doesn't think the entire file has changed.
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/7f6f7219-c9da-4a94-a158-bce6d367708d)

Reply:
Actually I will just do it in #261 and hopefully it will resolve itself.